tests/rdt/transformers/test_NumberTransformer.py

```diff
@@ -79,6 +79,29 @@ def test_reverse_transform(self):
         # Check
         assert result.equals(expected_result)
 
+    def test_reverse_transform_nan(self):
+        """FIXME: finish when number and null transformer are compatible"""
+        """ Checks the conversion of the data back into original format """
+
+        # Setup
+        col = pd.Series([34, 23, 27, 31, 39], name='age')
+        col_meta = {
+            'name': 'age',
+            'subtype': 'integer',
+            'type': 'number'
+        }
+        transformer = NumberTransformer()
+        transformer.fit_transform(col, col_meta, False)
+        col2 = pd.Series([34, 23, 27, 31, np.nan], name='age')
+
+        # Run
+        expected = pd.DataFrame({'age': [34, 23, 27, 31, transformer.default_val]})
+        print(expected)
```

Comment:
There should be any `print` or debbuging statements on the unittests.

Reply:
What I mean is that:
- If you were using the print statement for debugging, then it shouldn't have been commited.
- If you were using the print statement to check something, then it should changed to an assertion.

Reply:
Fixed by https://github.com/HDI-Project/RDT/pull/37/commits/de3e452437d1b4cc76a6888e9f95dd4304c8e118